Testscripts/Linux/STOR-Large-CopyFile.sh

```diff
@@ -0,0 +1,223 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+
+. utils.sh || {
+    echo "Error: unable to source utils.sh!"
+    exit 0
+}
+
+UtilsInit
+
+function Test_Local_Copy_File() {
+    LogMsg "Start to dd file"
+
+    #dd 5G files
+    dd if=/dev/zero of=/root/data bs=2048 count=2500000
```

Comment:
is it required to put the test file under /root?

Reply:
no